RecoTracker/LSTCore/src/alpaka/LST.dev.cc

```diff
@@ -94,22 +94,23 @@ void ALPAKA_ACCELERATOR_NAMESPACE::lst::LST::prepareInput(std::vector<float> con
       XYZVector p3PCA(see_px[iSeed], see_py[iSeed], see_pz[iSeed]);
       XYZVector r3PCA(calculateR3FromPCA(p3PCA, see_dxy[iSeed], see_dz[iSeed]));
 
-      float pixelSegmentDeltaPhiChange = (r3LH - p3LH_helper).phi();  //FIXME: this looks like a bug
+      // The charge could be used directly in the line below
+      float pixelSegmentDeltaPhiChange = ROOT::Math::VectorUtil::DeltaPhi(r3LH, p3LH);
```

Comment:
can it be that the sign is wrong in https://github.com/SegmentLinking/LSTGeometry/blob/15f482747748e89632fbef65989355ddbb8d94d7/DetectorGeometry.py#L157-L167 ?
B_z is positive, so `q  v x B` should push in negative phi direction; the values in the tracking ntuple seem to agree.
Also from the ntuple `deltaPhi(r_LH, p_LH)` matches with the charge quite well